packages/app/src/state/home/actions.ts

```diff
@@ -15,3 +16,13 @@ export const fetchOptimismMarkets = createAsyncThunk<
 	const serializedMarkets = serializeMarkets(markets)
 	return { markets: serializedMarkets }
 })
+
+export const fetchFuturesStats = createAsyncThunk<
+	Awaited<ReturnType<KwentaSDK['stats']['getFuturesStats']>>,
+	void,
+	ThunkConfig
+>('home/fetchFuturesStats', async (_, { extra: { sdk } }) => {
+	const stats = await sdk.stats.getFuturesStats()
```

Comment:
another place it would be good to catch and notify

Reply:
Added catch.